src/main/java/seedu/address/ui/EventCard.java

```diff
@@ -0,0 +1,66 @@
+package seedu.address.ui;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+import seedu.address.model.event.Event;
+
+/**
+ * An UI component that displays information of a {@code Person}.
+ */
+public class EventCard extends UiPart<Region> {
+
+    private static final String FXML = "EventListCard.fxml";
+
+    /**
+     * Note: Certain keywords such as "location" and "resources" are reserved keywords in JavaFX.
+     * As a consequence, UI elements' variable names cannot be set to such keywords
+     * or an exception will be thrown by JavaFX during runtime.
+     *
+     * @see <a href="https://github.com/se-edu/addressbook-level4/issues/336">The issue on AddressBook level 4</a>
+     */
+
+    public final Event event;
+
+    @FXML
+    private HBox cardPane;
+    @FXML
+    private Label name;
+    @FXML
+    private Label id;
+    @FXML
+    private Label dateTime;
+    @FXML
+    private Label description;
+
+    /**
+     * Creates a {@code PersonCode} with the given {@code Person} and index to display.
```

Comment:
Is there a mistake here? Not sure it should be `PersonCode`?

Reply:
Thank you for catching this!